example/lib/main.dart

```diff
@@ -114,6 +115,44 @@ class _MyAppState extends State<MyApp> {
     });
   }
 
+  void ctExperimentsUpdated() {
```

Comment:
@sarveshgk10 Please remove code related to Product Experience (Dynamic Vars) in the example project.